src/Workspaces/Core/Desktop/SymbolSearch/SymbolSearchUpdateEngineFactory.cs

```diff
@@ -76,7 +76,11 @@ private class RemoteUpdateEngine : ISymbolSearchUpdateEngine
                     _sessionDoNotAccessDirectly = null;
 
                     _client = await _workspace.GetRemoteHostClientAsync(CancellationToken.None).ConfigureAwait(false);
-                    _client.ConnectionChanged += OnConnectionChanged;
```

Comment:
are there other people that might observe that the client returned by GetRemoteHostClientAsync could be null?